### PR TITLE
{libqalculate, qalculate-gtk, qalculate-qt}: 4.8.1 -> 4.9.0

### PR DESCRIPTION
**Pull request by cyclic-pentane**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/science/math/qalculate-gtk/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   pname = "qalculate-gtk";
-  version = "4.8.1";
+  version = "4.9.0";
 
   src = fetchFromGitHub {
     owner = "qalculate";
     repo = "qalculate-gtk";
     rev = "v${finalAttrs.version}";
-    sha256 = "sha256-bG0hui5GjHWHny/8Rq5sZGz3s5rYnYlpc+K8I/LwDto=";
+    sha256 = "sha256-rQxOOxM4TazkDs/H3KEPbdo6WBl0ptyAlZwv8nnGMss=";
```
Comment (kirillrdy) [inline pkgs/applications/science/math/qalculate-gtk/default.nix:11]:
```suggestion
    hash = "sha256-rQxOOxM4TazkDs/H3KEPbdo6WBl0ptyAlZwv8nnGMss=";
```

**Comment by kirillrdy** [comment]
@alyaeanyx also gtk and qt derivations are missing `meta.mainProgram`

**Comment by alyaeanyx** [comment]
Done!

**Review by kirillrdy — APPROVED** [review]
Result of `nixpkgs-review pr 271142` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>56 packages built:</summary>
  <ul>
    <li>albert</li>
    <li>breath-theme</li>
    <li>cantor</li>
    <li>colloid-kde</li>
    <li>graphite-kde-theme</li>
    <li>labplot</li>
    <li>latte-dock</li>
    <li>layan-kde</li>
    <li>libqalculate</li>
    <li>libqalculate.dev</li>
    <li>libqalculate.doc</li>
    <li>libsForQt5.applet-window-appmenu</li>
    <li>libsForQt5.bismuth</li>
    <li>libsForQt5.kde-cli-tools</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.kdev-php</li>
    <li>libsForQt5.kdev-python</li>
    <li>libsForQt5.kdevelop</li>
    <li>libsForQt5.kdevelop-unwrapped</li>
    <li>libsForQt5.khotkeys</li>
    <li>libsForQt5.khotkeys.bin</li>
    <li>libsForQt5.khotkeys.dev</li>
    <li>libsForQt5.kinfocenter</li>
    <li>libsForQt5.kmenuedit</li>
    <li>libsForQt5.krohnkite</li>
    <li>libsForQt5.kwin-dynamic-workspaces</li>
    <li>libsForQt5.kwin-tiling</li>
    <li>libsForQt5.kzones</li>
    <li>libsForQt5.parachute</li>
    <li>libsForQt5.plasma-bigscreen</li>
    <li>libsForQt5.plasma-browser-integration</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5.plasma-disks</li>
    <li>libsForQt5.plasma-disks.bin</li>
    <li>libsForQt5.plasma-disks.dev</li>
    <li>libsForQt5.plasma-mobile</li>
    <li>libsForQt5.plasma-remotecontrollers</li>
    <li>libsForQt5.plasma-workspace</li>
    <li>libsForQt5.plasma-workspace.dev</li>
    <li>libsForQt5.powerdevil</li>
    <li>libsForQt5.systemsettings</li>
    <li>libsForQt5.systemsettings.bin</li>
    <li>libsForQt5.systemsettings.dev</li>
    <li>libsForQt5.xdg-desktop-portal-kde</li>
    <li>lightly-boehs</li>
    <li>lightly-qt</li>
    <li>nordic</li>
    <li>nordic.sddm</li>
    <li>pop-launcher</li>
    <li>qalculate-gtk</li>
    <li>qalculate-qt</li>
    <li>qogir-kde</li>
    <li>rofi-calc</li>
    <li>utterly-nord-plasma</li>
    <li>wacomtablet</li>
    <li>whitesur-kde</li>
  </ul>
</details>

**Comment by kirillrdy** [comment]
obforg eval error happens to other PRs

**Comment by doronbehar** [comment]
@ofborg eval

**Comment by doronbehar** [comment]
@GrahamcOfBorg eval

**Comment by doronbehar** [comment]
OK I'll take the almost empty risk of merging without waiting for ofborg - changes seem trivial after all and I can't trigger it.

**Pull request merged**